rl-institut/oemoflex
# Feature/GitHub actions

This PR resolves #14 and sets up github actions including:

     - run tests with pytest
     - add test coverage
     - check linting with black
     
`extras_require` are added to `setup.py`

Analogous to https://github.com/rl-institut/oemof-B3/pull/25 the checks are done for python 3.7 and 3.8.
